src/easypost.js

```diff
@@ -123,10 +126,16 @@ export default class API {
     } = params;
 
     const req = this.agent[method](this.buildPath(path))
-                  .accept('json')
-                  .set('Content-Type', 'application/json')
-                  .set(API.buildHeaders(headers))
-                  .auth(`${this.key}:`);
+                .accept('json')
+                .set('Content-Type', 'application/json')
+                .set(API.buildHeaders(headers))
+                .auth(`${this.key}:`);
+
+    if (path === 'rating/v1/rates') {
```

Comment:
Why does this need to be explicitly set? It should be hitting thus URL anyways, and the content-type is set to application/json above.